### ISSUE TITLE
Comments on articles

### ISSUE THREAD

**Issue by juancarmo** [fullstackworld/fullstackworld.com#17]
To make it more social, it would be a good idea to allow readers to leave comments on articles

**Comment by mhetreramesh** [comment]
Yes, this is in our roadmap next to login & posts from user. Let's put it in backlog for now.
